### ISSUE TITLE
add 10m resolution in a new release

### ISSUE THREAD

**Issue by silky** [etpinard/sane-topojson#6]
is it possible to get the 10m data in a new release? then we can use it in some other things, like plotly.

**Comment by etpinard** [comment]
Yeah. That would be nice :+1: 

I think all we need to do is add `10` to this [list](https://github.com/etpinard/sane-topojson/blob/3efc5797df570fc7143a121d0ca9903defb8a8d5/config.json#L4) and build the topojsons again.

**Comment by silky** [comment]
yes, that'll do for this library, but to use it in plotly.js, i think there's another list of 50,110 that we'll need to update

**Comment by etpinard** [comment]
Oh. You want to add that `10m` resolution in plotly.js maps. 

Oof, I'm not sure that's a good idea to be honest. At this resolution dragging and zooming interactions will be painfully slow. We could disable drag/zoom for the `10m`, but I'm not convinced that's the way to go.

**Comment by silky** [comment]
we needed better resolution over a certain area of australia

this seemed to me to be a reasonable way to achieve it.

maybe let's at least get them added in here, and then some thought can be given as-to how to do it in plotly.js

in particular, it doesn't seem unreasonable to, say, define a grid in which it renders the 10m res data, and outside of that it just reverts to 50 or 110.

the main goal is: get high quality coastal edges around a small city.
